docs/apache-airflow-providers-ssh/connections/ssh.rst

```diff
@@ -47,7 +47,8 @@ Extra (optional)
     * ``key_file`` - Full Path of the private SSH Key file that will be used to connect to the remote_host.
     * ``private_key`` - Content of the private key used to connect to the remote_host.
     * ``private_key_passphrase`` - Content of the private key passphrase used to decrypt the private key.
-    * ``timeout`` - An optional timeout (in seconds) for the TCP connect. Default is ``10``.
+    * ``conn_timeout`` - An optional timeout (in seconds) for the TCP connect. Default is ``10``. The paramater to :class:`~airflow.providers.ssh.hooks.ssh.SSHHook` conn_timeout takes precedence.
```

Comment:
Maybe we should document on the other side instead of here, like `remote_host`? Thoughts?

Reply:
Sorry - what do you mean by "on the other side"?

Reply:
My bad, I meant on the hook/operator side like was suggested for the operator in my other comment.